### PR TITLE
Prevent tap zoom as seen in https://media.dev.unee-t.com/2018-10-02/r…

### PR DESCRIPTION
**Pull request by kaihendry**
…eport-audio-fixed.mp4

### CODE REVIEW

File: client/main.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <head>
   <title>Unee-T</title>
-  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
```
Comment (nbiton) [inline client/main.html]: Shouldn't we keep `height=device-height`?

Comment (kaihendry) [reply client/main.html]: I shamelessly copied from view-source:https://busrouter.sg/ which behaves as I expect mobile Web app to do in Safari. i.e. double tap zoom is disabled.

Comment (nbiton) [reply client/main.html]: I'm halfway sure that it had a reason for being there. If remove it we need to verify our layouts still behave as before across all platforms

Comment (kaihendry) [reply client/main.html]: :man_shrugging:

**Comment by nbiton** [comment]
I did some more extensive testing on this on desktop and android, and it does look the same.

**Pull request merged**